app/controllers/stories/feeds_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Stories::FeedsController < ApplicationController
+  respond_to :json
+
+  def show
+    @stories = assign_feed_stories
+  end
+
+  private
+
+  def assign_feed_stories
```

Comment:
There some duplication here with `StoriesController` that I don't love, but it's slightly different.